src/helpers/print.js

```diff
@@ -94,11 +94,11 @@ const printPdf = () =>
       });
   });
 
-export const creatingPages = (pagesToPrint, includeComments, includeAnnot, printQualty, sortStrategy, clrMap, dateFormat, onProgress) => {
+export const creatingPages = (pagesToPrint, includeComments, includeAnnot, printQuality, sortStrategy, clrMap, dateFormat, onProgress) => {
   const createdPages = [];
   pendingCanvases = [];
   includeAnnotations = includeAnnot;
-  printQuality = printQualty;
+  PRINT_QUALITY = printQuality;
```

Comment:
Would it suffice to drop this line and simply have `multiplier: printQuality` in line 189?